src/DssDirectDepositHub.sol

```diff
@@ -0,0 +1,460 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+// Copyright (C) 2021-2022 Dai Foundation
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+pragma solidity 0.6.12;
+
+interface D3MPoolLike {
```

Comment:
Since this is not an external dependency we should import here.

`import { ID3MPool } from "../pools/ID3MPool.sol";

Reply:
In `dss` do we tend to declare `VatLike`, `VowLike`, etc..

Reply:
changed in 5945130